spec/factories/events.rb

```diff
@@ -123,6 +123,12 @@
       end
     end
 
+    trait :with_enough_free_places do
+      after(:build) do |event|
+        event.max_participants = event.application_letters.size + 1
```

Comment:
I disagree. If you got 0 application letters yet, you will set max_participants to 0, which is not a valid value